extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmReflections.java

```diff
@@ -1,161 +0,0 @@
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.CollectionFunction.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.BinaryLogicOperatorNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.CountNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.IsNullLogicOperatorNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.IdentNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.ComponentJoin.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.ParameterNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.AbstractSelectExpression.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.MapEntryNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.MapValueNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.InLogicOperatorNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.IndexNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.AbstractNullnessCheckNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.DotNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.ResultVariableRefNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.BetweenOperatorNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.AggregateNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.QueryNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.BooleanLiteralNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.SimpleCaseNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.AbstractMapComponentNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.FromReferenceNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.OrderByClause.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.FromClause.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.ConstructorNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.LiteralNode.class);
-        simpleConstructor(reflectiveClass, org.hibernate.hql.internal.ast.tree.BinaryArithmeticOperatorNode.class);
-        simpleConstructor(reflectiveClass, CommonToken.class);
```

Comment:
There's a native test failure that seems related to that line. Was it moved upstream too?

Reply:
thanks, looks like something is wrong.  The upstream metadata is rather different, I've been trying to optimise it further - I did run all Quarkus tests during its development, but looks like I screwed something up during the last polishing phase. I'll re-introduce this file so to add the missing entry.